components/brave_wallet/browser/filecoin_keyring.cc

```diff
@@ -36,7 +36,13 @@ bool GetBLSPublicKey(const std::vector<uint8_t>& private_key,
 }
 }  // namespace
 
-FilecoinKeyring::FilecoinKeyring() = default;
+FilecoinKeyring::FilecoinKeyring(const std::string& chain_id) {
+  auto* network = (chain_id == brave_wallet::mojom::kFilecoinMainnet)
```

Comment:
mojom::kFilecoinMainnet and mojom::kFilecoinTestnet are already a string, there is no point getting their pointer and dereferencing it

Reply:
done